src/main/java/karuna/karuna_backend/visitor/message/dto/VisitorMessageDtoList.java

```diff
@@ -0,0 +1,6 @@
+package karuna.karuna_backend.visitor.message.dto;
+
+import java.util.List;
+
+public record VisitorMessageDtoList(List<VisitorMessageDto> messagesDto) {
```

Comment:
I think in json i want to see message [] not messagesDto

Reply:
i will name it "messages"